### ISSUE TITLE
Image are not showing in smasung note 3

### ISSUE THREAD

**Issue by hussainahmad** [bumptech/glide#2984]
**Glide Version**: 3.7.0

**okhttp3 Integration**:1.4.0
**Device/Android Version**: fail on Samsung note 3 but working fine with A7, S7,S6 Note 5  
**Issue details / Repro steps / Use case background**: 
`Place Holder` image is only showing not the actual image from url 

**Glide load line / `GlideModule` (if any) / list Adapter code (if any)**:
```java
Glide.with(getActivity()).load(Item.getImg())
                    .diskCacheStrategy(DiskCacheStrategy.SOURCE)
                    .dontAnimate()
                    .into(imageView);
```

**Layout XML**:
```xml
 <ImageView
            android:id="@+id/imagView"
            android:layout_width="match_parent"
            android:layout_height="wrap_content"
            android:adjustViewBounds="true"
            android:background="#eeeeee" />
```

**Stack trace / LogCat**:
```logcat
V/Engine: Started new load in 0.053917ms, key: EngineKey{https://image_url.jpg+com.bumptech.glide.signature.EmptySignature@52b0b42c+[525x758]+''+'ImageVideoBitmapDecoder.com.bumptech.glide.load.resource.bitmap'+'FitCenter.com.bumptech.glide.load.resource.bitmap'+'BitmapEncoder.com.bumptech.glide.load.resource.bitmap'+'GifBitmapWrapperDrawableTranscoder.com.bumptech.glide.load.resource.transcode'+''}
V/DecodeJob: Decoded transformed from cache in 0.106996, key: EngineKey{https://image_url.jpg+com.bumptech.glide.signature.EmptySignature@52b0b42c+[525x758]+''+'ImageVideoBitmapDecoder.com.bumptech.glide.load.resource.bitmap'+'FitCenter.com.bumptech.glide.load.resource.bitmap'+'BitmapEncoder.com.bumptech.glide.load.resource.bitmap'+'GifBitmapWrapperDrawableTranscoder.com.bumptech.glide.load.resource.transcode'+''}

```

**Note:** Issue is coming only in selected device, in all other device image loading working fine

**Comment by sjudd** [comment]
Let me know if you can reproduce on Glide v4+

**Comment by hussainahmad** [comment]
Hi @sjudd , Sorry  Issue is due `SSLProtocolException`  which is already discussed in #1679 , after updating `UnsafeOkHttpClient` Issue resolved by using square/okhttp#2372
